feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -275,73 +283,96 @@ class PreviewViewModel @AssistedInject constructor(
     }
 
     private fun getCaptureToggleUiStateDisabledReason(
```

Comment:
Does the toggle UI component need to be visible when the concurrent camera is on? Its purpose *looks* a little confusing since it isnt present otherwise

Reply:
This is currently the only UI affordance we have to show that only video capture is supported. It also gives the user something to click on to get the reason why image capture is disabled.

It's not the greatest UX. Ideally we should add some additional UI changes to the capture button make it clearer that it will not capture images and will only start a video recording.